public/locales/ru/paper-wallet.json

```diff
@@ -0,0 +1,18 @@
+{
+  "connect": {
+    "title": "Подключите ваш кошелек Tari Universe к телефону",
+    "text": "Скачайте Tari Aurora и отслеживайте свои XTM в любом месте. Получите его сейчас в App Store или Google Play Store.",
```

Comment:
For sentence "**Получите** его сейчас в App Store или Google Play Store.". Instead of **Получите** could be more correct to use words **Скачайте** or **Загрузите**

Reply:
```suggestion
    "text": "Скачайте Tari Aurora и отслеживайте свои XTM в любом месте. Загрузите его сейчас в App Store или Google Play Store.",
```